src/bufr/IodaEncoder/IodaEncoder.cpp

```diff
@@ -8,6 +8,7 @@
 #include "IodaEncoder.h"
 
 #include <memory>
+#include <type_traits>
```

Comment:
Not sure you need to include this here... I would try compiling without it.

Reply:
I don't know, maybe something from the previous commit not purged.

Reply:
If you need the include its fine to leave it in. I was just wondering if it could be removed, as I don't like having unused includes in files.